Loggie/Classes/LogDetails/LogDetailsViewController.swift

```diff
@@ -130,3 +130,59 @@ extension LogDetailsViewController: UITableViewDataSource {
         }
     }
 }
+
+extension LogDetailsViewController: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.addAction(copyAction(for: indexPath))
+        actionSheet.addAction(shareAction(for: indexPath))
```

Comment:
Please add `.cancel` action as well (and handler for deselecting the row as well) otherwise you cannot dismiss alert if you pressed some cell accidentally.

Reply:
It is an action sheet and any tap on the screen will dismiss it. Anyway, I added cancel action.